hr_expense_invoice/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Set supplier invoices on HR expenses
+====================================
+
+This module allows to set a supplier invoice for each expense line, adding
+the corresponding journal items to transfer the debt to the employee.
```

Comment:
This module should be used when a supplier invoice is paid by an employee. It allows to set ...

:+1: if you add that to the description.